cdb2api/cdb2api.c

```diff
@@ -4432,13 +4461,18 @@ static int cdb2_get_dbhosts(cdb2_hndl_tp *hndl)
     }
 
 retry:
-    if (rc && num_retry < MAX_RETRIES) {
+    if (rc) { 
+        if(num_retry >= MAX_RETRIES)
+            return rc;
+
         num_retry++;
         poll(NULL, 0, 250); // Sleep for 250ms everytime and total of 5 seconds
         rc = 0;
-    } else if (rc) {
-        return rc;
     }
+    if (hndl->debug_trace)
+        fprintf(stderr, "td %d %s line %d: get blah num_retry=%d\n",
```

Comment:
What is `blah`?

Reply:
function name, obviously :) 
clearly I need to clean it up.
thanks